### PR TITLE
Read only user restrictions

### LINKED ISSUE #298: Hiding periods from period dropdown
On accountperiod.edit page we should have a checkbox next to each period which will indicate if that period will be shown in the regular period dropdown select for users with access level one(only read), one more field in DB table for account period.
This should affect the global function for creating the dropdown.

### LINKED ISSUE #299: Hide some information from users with access level one (read only)
On salary.edit and invoice.edit page if the user has access level one (read only) they should not be able to see the information on who created/updated/sent the salary/invoice and when.

### PR DESCRIPTION
**Pull request by mladjo2505**
Closes #298 
Closes #299 
And some additional restrictions to what a read only user can do.

### CODE REVIEW

File: modules/accountperiod/view/edit.php [hunk 1]
```diff
@@ -40,14 +40,15 @@
...
-      <td colspan="6" align="right"><? print $_lib['form3']->Input(array('type'=>'submit', 'name'=>'action_accountperiod_update', 'value'=>'Lagre periode (S)', 'accesskey' => 'S')) ?></td>
+      <td colspan="<? print (($_lib['sess']->get_person('AccessLevel') > 1) ? 7 : 6); ?>" align="right"><? print $_lib['form3']->Input(array('type'=>'submit', 'name'=>'action_accountperiod_update', 'value'=>'Lagre periode (S)', 'accesskey' => 'S')) ?></td>
     </tr>
   <tr>
     <th>Periode</th>
     <th>Lukket</th>
     <th>�pen</th>
     <th>Ferdig</th>
     <th>Stengt</th>
+    <? if ($_lib['sess']->get_person('AccessLevel') > 1) { print '<th>F&oslash;r lese</th>'; } ?>
```
Comment (mladjo2505) [inline modules/accountperiod/view/edit.php]: Før --> Får

**Comment by mladjo2505** [comment]
Last commit by @hildegunn95 checked by @mladjo2505

**Comment by mladjo2505** [comment]
FINAL REVIEW QUESTION:
@stoivo Should the 'choose year' dropdown be affected by this?

**Comment by sls-lnluksa** [comment]
First round review: Nikola Lukovic (sls-lnluksa)
Final review with: @stoivo

**Pull request merged**